### PR TITLE
Update dg user stories

### PR DESCRIPTION
**Pull request by yucheng11122017**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -831,7 +815,89 @@ Future versions user stories
...
+   Use case ends.
+
+**Extensions**
+
+* 1a. There is no previous command history.
+    * 1a1. CONNETIONS will not display command.
+
+      Use case ends.
+
 *{More to be added}*
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:901]: Can remove this

File: docs/UserGuide.md [hunk 2]
```diff
@@ -341,7 +342,6 @@ Exports a CSV file of the current view containing specified fields.
 Notes:
 * Name is always the first column in the CSV file.
 * The default exported fields are name, phone, email.
-* Invalid prefixes are ignored.
```
Comment (Jiwei99) [inline docs/UserGuide.md:344]: Is this part onwards a duplicate of the other other PR #182 ?

Comment (yucheng11122017) [reply docs/UserGuide.md:344]: Oh sorry my bad!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -2,8 +2,28 @@
...
+* [Implementation](#implementation)
+    * [Tags](#tags)
+    * [Pin feature](#pin-feature) 
+    * [Find feature](#find-feature)
+    * [FindAny feature](#findany-feature)
+    * [Help feature](#help-feature)
+    * [Birthday Reminder feature](#birthday-reminder-feature)
+    * [Mailing List feature](#mailing-list-feature)
+    
+* [Documentation, logging, testing, configuration, dev-ops](#documentation-logging-testing-configuration-dev-ops)
```
[the remark this answers is not in the document]
Comment (yucheng11122017) [reply docs/DeveloperGuide.md:24]: Is guides a bit too vague though? I copied this from the original DG haha

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -463,63 +483,27 @@ Step 3. CONNECTIONS will not display the first person's invalid `Tag` and will n
 
 ### User stories
 
-Chosen user stories v1.2
 
 | As a... | I want to... | So that I can... |
 |---|---|---|
-| Person who loves Birthdays    | Track birthdays                       | Easy way of checking the birthday of friends
-| Sociable person               | Partition frequent contacts           | Easy to access people within each group
+| Person who loves Birthdays    | Track birthdays                       | Easily checking the birthday of friends
```
[the remark this answers is not in the document]
Comment (yucheng11122017) [reply docs/DeveloperGuide.md:489]: rip me

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -463,63 +483,27 @@ Step 3. CONNECTIONS will not display the first person's invalid `Tag` and will n
...
+| Person who Loves Birthdays and is forgetful    | have birthday reminders            | Avoid miss any of his/her friends’ birthdays
```
[the remark this answers is not in the document]
Comment (yucheng11122017) [reply docs/DeveloperGuide.md:490]: right thanks!

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -463,63 +483,27 @@ Step 3. CONNECTIONS will not display the first person's invalid `Tag` and will n
...
+| Party Organiser       | Look for contacts details for all my friends      | Send out my invites
+| Party Organiser       | Generate csv file of my invitees                  | Take attendance or make external notes
 | Party Organiser               | Add people to a party list            | Easily extend a party invite list
```
[the remark this answers is not in the document]
Comment (yucheng11122017) [reply docs/DeveloperGuide.md:493]: I thought it is like the csv file but yeah a bit vague ill delete bah

[the remark this answers is not in the document]
Comment (yucheng11122017) [reply docs/DeveloperGuide.md:493]: Ill change to be more explicit i think!

File: docs/UserGuide.md (hunk 2 above)
Comment (yucheng11122017) [reply docs/UserGuide.md:344]: Ehh sorry my bad!

File: docs/DeveloperGuide.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jiwei99) [reply docs/DeveloperGuide.md:24]: Ah okie then

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -463,63 +483,28 @@ Step 3. CONNECTIONS will not display the first person's invalid `Tag` and will n
...
-| Experienced user      | Quickly add contact without my mouse              | Just type
-| Experienced user      | Get suggestions on commonly run commands          | don't have to keep typing the commands they use frequently.
-|  |  |  |
-| Forgetful person      | Use the help button                               | Remind myself how to use the app
-| Forgetful person      | Select from existing tags                         | Have consistent labelling
-|  |  |  |
-| Party Organiser       | Look for contacts details for all my friends      | Send out my invites
-| Party Organiser       | Generate mailing list from my contacts            | Contact all invitees as a group
+| Person who loves birthdays    | Track birthdays                       | Easily check the birthday of friends
+| Person who Loves birthdays and is forgetful    | have birthday reminders            | Avoid missing any of his/her friends’ birthdays
```
Comment (Jiwei99) [inline docs/DeveloperGuide.md:490]: `loves` small caps also lol

**Review by leezhixuan — APPROVED** [review]
LGTM 👍

File: docs/UserGuide.md (hunk 2 above)
Comment (Jiwei99) [reply docs/UserGuide.md:344]: J make sure if it merges it won't affect the other PR bah

**Review by Jiwei99 — APPROVED** [review]
LGTM

**Pull request merged**